hawkbit-runtime/.sandbox/hawkbit-sandbox-cleanup-task.sh

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2018 Bosch Software Innovations GmbH and others.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+
+STACK_NAME=hawkbit
+
+echo "/------------------------------------------------------------------------------"
+echo "|  Data Clean Up START:  $(date -u) "
+echo "+------------------------------------------------------------------------------"
+echo "|"
+echo "+-- Remove Docker Stack:"
+docker stack rm $STACK_NAME
+sleep 15
+echo "|"
+echo "+-- Remove Docker Container:"
+docker container prune --force
```

Comment:
What about `docker system prune --all --force --volumes` here instead of single commands?